test/extended/setup.sh

```diff
@@ -2,6 +2,8 @@
 #
 # This abstracts starting up an extended server.
 
+source "${OS_ROOT}/hack/util.sh"
```

Comment:
@stevekuznetsov oh ... it's clear now. The reason, why `cleanup_openshift` wasn't defined of the test [job](https://ci.openshift.redhat.com/jenkins/job/test_pull_requests_origin_conformance/8838/consoleFull) is that the tested [PR](https://github.com/openshift/origin/pull/11838) remamed it to `os::cleanup::openshift`.

So this PR doesn't have any value. I'm just trying to reproduce to the #12007 here.